varcode/coding_effect.py

```diff
@@ -32,12 +35,28 @@
 from .mutate import substitute, insert_after
 from .string_helpers import trim_shared_flanking_strings
 
-from Bio.Seq import Seq
-from Bio.Data import CodonTable
-
+DNA_CODON_TABLE = CodonTable.standard_dna_table.forward_table
 START_CODONS = set(CodonTable.standard_dna_table.start_codons)
 STOP_CODONS = set(CodonTable.standard_dna_table.stop_codons)
 
+def translate_codon(codon, aa_pos):
+    """Translate a single codon into a single amino acid or stop '*'
+
+    Parameters
+    ----------
+    codon : str
+        Expected to be of length 3
+    aa_pos : int
+        Codon/amino acid offset into the protein (starting from 0)
+    """
+    # not handling rare Leucine or Valine starts!
+    if aa_pos == 0 and codon in START_CODONS:
+        return "M"
+    elif codon in STOP_CODONS:
+        return "*"
+    else:
+        return DNA_CODON_TABLE[codon]
+
 def translate(cds_seq):
```

Comment:
super nitpicky nit: maybe rename `cds_seq` to something a little clearer like `coding_sequence` or even just `cds`.  If `cds` stand for "coding sequence", then `cds_seq` would be a sequence of coding sequences?

Also perhaps clarify the significance of it being a `cDNA coding sequence` in the comment? Does that mean that it's the reverse complement of the mRNA transcript, or is that meant to imply some more subtle restriction on what I can pass to this function?